plank/scripts/create-selfsigned-cert.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
```

Comment:
we could certainly carve out a separate section in README but i figured just running this once and for all without following commands in the README would be a better no brainer way to go about it. also there's really not much to maintain other than re-running the script to regenerate a pair of cert and private key in case the cert has expired.